cryptol/CheckExercises.hs

```diff
@@ -331,7 +331,7 @@ main = do
 
               writeFile outFile outText
 
-              let diffCmd = (P.shell ("diff " ++ outExpectedFile ++ " " ++ outFile))
+              let diffCmd = (P.shell ("diff -u " ++ outExpectedFile ++ " " ++ outFile))
```

Comment:
This makes the diff output in the more familiar git style, if you prefer it the other way I can drop this change ^^